examples/temperature-measurement-app/esp32/main/temperature-measurement.zap

```diff
@@ -0,0 +1,1171 @@
+{
+  "writeTime": "Tue Nov 17 2020 19:19:54 GMT+0100 (Central European Standard Time)",
+  "featureLevel": 4,
+  "creator": "zap",
+  "keyValuePairs": [
+    {
+      "key": "commandDiscovery",
+      "value": "1"
+    },
+    {
+      "key": "defaultResponsePolicy",
+      "value": "always"
+    },
+    {
+      "key": "manufacturerCodes",
+      "value": "0x1002"
+    }
+  ],
+  "package": [
+    {
+      "pathRelativity": "relativeToZap",
+      "path": "../../../../../third_party/zap/repo/zcl-builtin/silabs/zcl.json",
+      "version": "ZCL Test Data",
+      "type": "zcl-properties"
+    },
+    {
+      "pathRelativity": "relativeToZap",
+      "path": "../../../../../src/app/zap-templates/chip-templates.json",
+      "version": "chip-v1",
+      "type": "gen-templates-json"
+    }
+  ],
+  "endpointTypes": [
+    {
+      "name": "Anonymous Endpoint Type",
+      "deviceTypeName": "CBA-tempsensor",
+      "deviceTypeCode": "0x0302",
+      "deviceTypeProfileId": "0x105",
+      "clusters": [
```

Comment:
Hmm.  This has Basic client and Basic server on the same endpoint.  Is that expected?

Reply:
That's because the file format is misleading and contains a lot of noises sadly.

In fact all the clusters you have mentioned appears in this configuration file because there are part of the default set of clusters for a regular "CBA-tempsensor".

But in this particular configuration file, most clusters are not enabled. There is a json field named `"enabled"` that controls if a particular cluster is enabled by setting it to `1` or `0`.

The only clusters that are enabled for this configuration file are the one that are used in the current `temperature-measurement-app`:
 * Basic Cluster Server
 * Temperature Measurement Server